packages/reports/addon/components/report-actions/multiple-format-export.js

```diff
@@ -122,16 +123,24 @@ export default class MultipleFormatExport extends Component {
    * A hack to make the trigger responding to click
    */
   @action
-  open() {
-    return true;
+  open(dropdown) {
+    if (this.closeTimer) {
+      cancel(this.closeTimer);
+      this.closeTimer = null;
+    } else {
+      dropdown.actions.open();
+    }
   }
 
   /**
    * @action close
    */
   @action
   close(dropdown) {
-    dropdown.actions.close();
+    this.closeTimer = later(() => {
+      this.closeTimer = null;
+      dropdown.actions.close();
+    }, 300);
```

Comment:
What's happening here?

Reply:
This is moving https://github.com/yavin-dev/framework/pull/1343 forward so it's replacing the `closeDelay` behavior from BasicDropdownHover

Reply:
got it